.vscode/settings.json

```
@@ -0,0 +1,14 @@
+{
+  // https://squidfunk.github.io/mkdocs-material/creating-your-site/
+  "yaml.schemas": {
+    "https://squidfunk.github.io/mkdocs-material/schema.json": "mkdocs.yml"
```

NIT: Spell check. "mkdocs.yaml" 
```suggestion
    "https://squidfunk.github.io/mkdocs-material/schema.json": "mkdocs.yaml"
```

Or if `yml` is preffered then modify the file extension

---

Well spotted! Thank You for noticing this oversight. Done / fixed; resolving.